### DISCUSSION TITLE
chaining pseudo-classes (placeholder:first-letter:uppercasse)

### DISCUSSION THREAD

**Discussion by Tylerpfarris** [tailwindlabs/tailwindcss#12899] [Help] [answered]
Hey yall!

As the title implies I'm curious about chaining off of the placeholder pseudo element. Im my case, I'm trying to capitalize the first letter of an inputs placeholder text with no success.

`<input className="placeholder:first-letter:uppercase">`

Just curious if this is possible, and if not why.

Thanks!

**Comment by adamwathan** [comment, marked as answer]
Hey! So the reason for this is actually just a CSS limitation — almost all pseudo-elements in CSS are just not stackable with each other, the browser just doesn't let you do it.

The same is true even for pseudo-classes on pseudo-elements — you can't do `.my-class::before:hover` for example.

So not a Tailwind limitation here, just a limitation of what browsers support even with vanilla CSS I'm afraid.

**Reply by Tylerpfarris** [reply]
Ahh i was not aware of that. Thanks for the clarification// lesson Adam, most appreciated!
